src/parakeet/Driver.cpp

```diff
@@ -399,6 +383,7 @@ namespace parakeet
         double startAngle = data->from / 10.0;
         double endAngle = (static_cast<double>(data->from) + static_cast<double>(data->span)) / 10.0;
         double anglePerPoint = (endAngle - startAngle) / data->count;
```

Comment:
Should probably update these to startAngle_deg, etc.

Reply:
Done.